cocos/tween/tween-action.ts

```diff
@@ -213,7 +213,7 @@ export class TweenAction<T> extends ActionInterval {
     }
 
     update (t: number): void {
-        const workerTarget = this.workerTarget ?? this.target;
+        const workerTarget = (this.workerTarget ?? this.target) as T;
```

Comment:
Fix: Error: ../cocos/tween/tween-action.ts:250:13 - error TS2[57](https://github.com/cocos/cocos-engine/actions/runs/9282363618/job/25540217690?pr=17061#step:12:58)1: Object is of type 'unknown'.
250             workerTarget[name] = prop.current;
                ~~~~~~~~~~~~